### PR TITLE
FEATURE: Switch between sites without logging in again

### PR DESCRIPTION
**Pull request by kdambekalns**
This eliminates the need for logging in repeatedly when switching between sites in a multi-site
setup.

NEOS-248 #close Fixes the issue
NEOS-240 #comment Solves part two of this issue

### CODE REVIEW

File: TYPO3.Neos/Classes/TYPO3/Neos/Controller/LoginController.php [hunk 1]
```diff
@@ -105,6 +119,27 @@ public function indexAction($username = null, $unauthorized = false)
     }
 
     /**
+     * Logs a use in if a session identifier is available under the given token in the token cache.
```
Comment (daniellienert) [inline TYPO3.Neos/Classes/TYPO3/Neos/Controller/LoginController.php]: should be "user"

File: TYPO3.Neos/Classes/TYPO3/Neos/Controller/Backend/BackendController.php [hunk 2]
```diff
@@ -48,6 +62,33 @@ public function indexAction()
     }
 
     /**
+     *
```
Comment (daniellienert) [inline TYPO3.Neos/Classes/TYPO3/Neos/Controller/Backend/BackendController.php]: Can be removed

**Comment by daniellienert** [comment]
Thanks @kdambekalns, the editors will love it :) Only found some nitpicks in the comments.
:+1: by reading.

**Comment by skurfuerst** [comment]
:+1: by reading -- really cool! Merging with previous votes.

**Pull request merged**